lisa/sut_orchestrator/azure/features.py

```diff
@@ -1957,6 +1949,7 @@ def _call_requirement_method(self, method_name: str, capability: Any) -> Any:
             capability, ArchitectureSettings
         ), f"actual: {type(capability)}"
 
+        assert self.arch == capability.arch, "req and capability should be the same"
```

Comment:
Use aseert_that to print original values

Reply:
updated